src/app/legacy/psammead/psammead-timestamp/src/index.jsx

```diff
@@ -29,39 +27,23 @@ const StyledTimestamp = styled.time`
 const Timestamp = ({
   children,
   datetime,
-  typographyFunc,
+  typographyFunc = getBrevier,
   script,
-  padding,
+  padding = true,
   service,
-  className,
+  className = '',
```

Comment:
```suggestion
  className = null,
```

Reply:
As previously mentioned, TS will complain in TSX components importing JSX if the values supplied mismatch.